modules/subchannel/src/meshgenerators/SCMDetailedQuadSubChannelMeshGenerator.C

```diff
@@ -33,9 +33,12 @@ SCMDetailedQuadSubChannelMeshGenerator::validParams()
   params.addRequiredParam<unsigned int>("n_cells", "The number of cells in the axial direction");
   params.addRequiredParam<unsigned int>("nx", "Number of channels in the x direction [-]");
   params.addRequiredParam<unsigned int>("ny", "Number of channels in the y direction [-]");
-  params.addRequiredParam<Real>("gap",
-                                "(Its an added distance between a perimetric pin and the duct: "
-                                "Edge Pitch W = pitch/2 - pin_diameter/2 + gap) [m]");
+  params.addRequiredParam<Real>(
+      "gap",
+      "(The side gap, not to be confused with the gap between pins, this refers to the gap "
+      "next to the duct or else the distance between the subchannel centroid to the duct wall."
+      "Edge Pitch W = (pitch/2 - pin_diameter/2 + gap) [m]");
```

Comment:
should this be side_gap?

What is "edge pitch W" ?

Reply:
this is not really clearer. 
Maybe write 
distance(edge pin center, duct wall) = pitch / 2 + side_gap

where is the subchannel centroid? You mean the coolant channel?